cockpit/src/subscriptions-view.jsx

```diff
@@ -257,22 +303,58 @@ class SubscriptionStatus extends React.Component {
             );
         }
 
-        let label;
+        let status_text;
         let action;
+        let org_name;
+        let sca_mode;
+        let status_element;
 
         if (this.props.status === 'unknown') {
-            label = _("Not registered");
+            status_text = _("Not registered");
             action = (
                 <Button onClick={this.handleRegisterSystem}>{_("Register")}</Button>
             );
         } else {
             const isUnregistering = (this.props.status === "unregistering");
-            label = this.props.status_msg;
+            status_text = this.props.status_msg;
             action = (
                 <Button isDisabled={isUnregistering} isLoading={isUnregistering}
                               onClick={this.handleUnregisterSystem}>{isUnregistering ? _("Unregistering"): _("Unregister")}</Button>
             );
         }
+
+        sca_mode = false;
+        if (this.props.org === undefined) {
+            org_name = '';
+        } else {
+            // Organization name
+            if ('displayName' in this.props.org) {
+                org_name = this.props.org['displayName'];
+            } else {
+                org_name = '';
+            }
+            // SCA mode tooltip
+            if ('contentAccessMode' in this.props.org) {
+                if (this.props.org['contentAccessMode'] === 'org_environment') {
+                    sca_mode = true;
+                }
+            }
+        }
+
+        // Display tooltip over "Disabled" with simple explanation, when SCA mode is used
+        if (sca_mode) {
+            const status_tooltip = (
+                <div>{_("Content Access Mode is set to Simple Content Access. This host has access to content, regardless of subscription status.")}</div>
+            );
+            status_element = (
+                <Tooltip content={status_tooltip} isContentLeftAligned='true' position='right'>
```

Comment:
Proposal: unless the user hovers over the disabled element - which probably they won't do because it's not a link or something - they won't see the tooltip

Maybe would be better to have a popover with an info icon? Like in  https://www.patternfly.org/v4/components/form/#basic

Reply:
Or a 'show more' link next to the 'Disabled' string.

Reply:
I was also thinking about `Popup`. I will do some experiments.